src/pseudopeople/noise_functions.py

```diff
@@ -20,7 +22,9 @@ def omit_rows(
 
 
 def duplicate_rows(
-    form_data: pd.DataFrame, configuration: float, randomness_stream: RandomnessStream
+    form_data: pd.DataFrame,
+    configuration: ConfigTree,
```

Comment:
These row noise types just take a float value, since `configuration.form.row_noise_type` is a value, not an internal node of the ConfigTree.